Grading-and-Student-Progress.md

```diff
@@ -44,10 +44,12 @@ Graded assignment scores are recorded in Canvas.
    1. Work will be in repl.it classroom or github
    1. A live demo may also be required depending on assignment instructions
    1. When in repl.it classroom, you will need to click *Mark complete* if the code meets all requirements
+   1. If the solution comes with a rubric, use the rubric to double check that the students solution meets all of the requirements. If something is missing, do NOT use the rubric as feedback. Tell the student directly what elements are missing.
```

Comment:
Typo: "...that the student's solution..."

Reply:
Fixed!